### PR TITLE
Sdes 216

### PR DESCRIPTION
**Pull request by JanosGit**

### CODE REVIEW

File: include/vctr/Expressions/Readme.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ the expression:
 - SSE: Implement `SSERegister<value_type> getSSE (size_t i) const` for suitable types. Make sure to require at
   least `archX64 && has::getSSE<SrcType>`
 - Neon: Implement `NeonRegister<value_type> getNeon (size_t i) const` for suitable types. Make sure to require at
-  least `archARM && has::getAVX<SrcType>`
+  least `archARM && has::getNeon<SrcType>`
```
Comment (iromur) [inline include/vctr/Expressions/Readme.md:186]: yuk…

**Review by iromur — APPROVED** [review]
Looks god and evident!

File: include/vctr/Containers/Span.h [hunk 2]
```diff
@@ -102,6 +102,14 @@ class Span : public VctrBase<ElementType, std::span<ElementType, extent>, extent
         VCTR_ASSERT (extent == containerToView.size() || extent == std::dynamic_extent);
     }
 
+    /** Creates a Span that acts as a view to a compile time constant sized C style array. */
+    template <size_t n>
+    requires (n != std::dynamic_extent)
+    constexpr Span (ElementType (&cStyleArray)[n])
```
Comment (p4th4x0r) [inline include/vctr/Containers/Span.h]: any test for that?

**Review by p4th4x0r — APPROVED** [review]

File: include/vctr/Expressions/Readme.md (hunk 1 above)
Comment (JanosGit) [reply include/vctr/Expressions/Readme.md:186]: Fortunately only in the documentation ;)

File: include/vctr/Containers/Span.h (hunk 2 above)
Comment (JanosGit) [reply include/vctr/Containers/Span.h]: Of course! Have a look at the changes in test/TestCases/SpanConstructors.cpp

**Pull request merged**